imports/plugins/core/dashboard/client/templates/shop/settings/settings.html

```diff
@@ -102,6 +102,7 @@
             {{> afQuickField name='settings.openexchangerates.refreshPeriod' placeholder="every 1 hour"}}
             {{> afQuickField name='settings.google.clientId'}}
             {{> afQuickField name='settings.google.apiKey'}}
+            {{> afQuickField name='settings.cart.cleanupDurationDays' placeholder="Every 3 days"}}
```

Comment:
Unless I am misunderstanding this is a duration, but it's stated as a schedule i.e. "every 3 days". This job runs once a day regardless correct? So this should be "Older than X days" or something similar.